src/validations/test/ssp.xspec

```diff
@@ -4364,6 +4364,283 @@
+                                            <value>[replace with list of personnel or roles]</value>
+                                        </set-parameter>
+                                        <set-parameter param-id="ac-1_prm_2">
+                                            <value>at least every 3 years</value>
+                                        </set-parameter>
+                                        <set-parameter param-id="ac-1_prm_3">
+                                            <value>at least annually</value>
+                                        </set-parameter>
+                                    </by-component>
+                                </statement>
+                            </implemented-requirement>
+                            <implemented-requirement control-id="ma-1" uuid="f0c6b63f-6b94-448f-bb16-db3d54b91734">
+                                <prop ns="https://fedramp.gov/ns/oscal" name="planned-completion-date" value="2020-11-27Z"/>
+                                <prop ns="https://fedramp.gov/ns/oscal" name="implementation-status" value="planned">
+                                    <remarks>
+                                        <p>Describe the plan to complete the implementation.</p>
+                                    </remarks>
+                                </prop>
+                                <prop ns="https://fedramp.gov/ns/oscal" name="control-origination" value="sp-system"/>
+                                <responsible-role role-id="program-director"/>
+                                <statement statement-id="ma-1_smt" uuid="36fd8d6c-4d47-46af-a5be-fca3c0fa143c">
+                                    <by-component component-uuid="60f92bcf-f353-4236-9803-2a5d417555f4"
+                                        uuid="82d21629-d0e1-40cd-955f-5fccd7bb652a">
+                                        <description>
+                                            <p>Describe how the control is satisfied within the system.</p>
+                                        </description>
+                                        <set-parameter param-id="ma-1_prm_1">
+                                            <value>organization-defined personnel or roles</value>
+                                        </set-parameter>
+                                        <set-parameter param-id="ma-1_prm_2">
+                                            <value>[defined the length of time]</value>
+                                        </set-parameter>
+                                        <set-parameter param-id="ma-1_prm_3">
+                                            <value>at least annually</value>
+                                        </set-parameter>
+                                    </by-component>
+                                </statement>
+                            </implemented-requirement>
+                        </control-implementation>
```

Comment:
This tests `set-parameter` within `implemented-requirement` but not when `set-parameter` is a child of `implemented-requirement`.